### PR TITLE
Adding packit package

### PR DESCRIPTION
**Pull request by r78v10a07**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/packit/version.patch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--- setup.py	2019-06-17 17:23:14.000000000 -0400
++++ setup.py.new	2019-07-31 14:36:01.000000000 -0400
+@@ -1,5 +1,5 @@
+ from setuptools import setup
+
+
+-setup(setup_requires=['pbr>=4.0', 'wheel>=0.24', 'glob2==0.4.1', 'packaging==16.8'], pbr=True, long_description_content_type="text/markdown")
++setup(setup_requires=['pbr>=4.0', 'wheel>=0.24', 'glob2>=0.4', 'packaging>=16.8'], pbr=True, long_description_content_type="text/markdown")
```
Comment (marcelotrevisani) [inline recipes/packit/version.patch]: I think you should raise an error on this project about it the ``glob2`` and ``packaging``

Comment (marcelotrevisani) [reply recipes/packit/version.patch]: And ``glob2>=0.4.1`` should be like that right?

File: recipes/packit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - pip
+    - setuptools
+    - wheel>=0.24
+    - pbr>=4.0
+    - glob2>=0.4.1
+    - packaging>=16.8
+
+  run:
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/packit/meta.yaml]: Do you need ``setuptools`` to run this project?

File: recipes/packit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/58/4f/0ac86eb3cc9ce74f87de314faf33426798052b5b17a8e18bb4f0a08686b4/packit-0.24.tar.gz
+  sha256: '{{ hash_value }}'
+  patches:
+    - version.patch
+
+build:
+  skip: true  # [win]
```
Comment (marcelotrevisani) [inline recipes/packit/meta.yaml]: I think this recipe can be ``noarch: python`` right?
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

File: recipes/packit/version.patch (hunk 1 above)
Comment (r78v10a07) [reply recipes/packit/version.patch]: Yes, you're right

File: recipes/packit/meta.yaml (hunk 3 above)
Comment (r78v10a07) [reply recipes/packit/meta.yaml]: You're right

File: recipes/packit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - wheel>=0.24
+    - pbr>=4.0
+    - glob2>=0.4.1
+    - packaging>=16.8
+
+  run:
+    - python
+    - setuptools
+    - wheel>=0.24
```
Comment (marcelotrevisani) [inline recipes/packit/meta.yaml]: It needs a space before the >=

File: recipes/packit/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - python
+    - pip
+    - wheel >=0.24
+    - pbr >=4.0
+    - glob2 >=0.4.1
+    - packaging >=16.8
+
+  run:
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/packit/meta.yaml]: Does this project rely on ``setuptool`` to run or it is necessary just during installation?

**Comment by marcelotrevisani** [comment]
ping @r78v10a07

File: recipes/packit/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
+{% set version = "0.24" %}
+{% set hash_value = "f22cbf399a5fcded061c566691be58d421b6c2b668ee0602665749a0266715f7" %}
+
+package:
+  name: python-packit
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/58/4f/0ac86eb3cc9ce74f87de314faf33426798052b5b17a8e18bb4f0a08686b4/packit-0.24.tar.gz
```
Comment (marcelotrevisani) [inline recipes/packit/meta.yaml]:
```suggestion
  url: https://files.pythonhosted.org/packages/58/4f/0ac86eb3cc9ce74f87de314faf33426798052b5b17a8e18bb4f0a08686b4/packit-{{ version }}.tar.gz
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**